### PR TITLE
Add Pet Store Fixure

### LINKED ISSUE #12: Add pet store example fixture

### PR DESCRIPTION
**Pull request by daveshah**
Closes #12

### CODE REVIEW

**Comment by daveshah** [comment]
@conzett I was wondering if, as part of this, we should reference this (the fixture data) in #7 so others can see how to reference seed data? If so, I'll wait until that's merged in and reference this in a test.

**Comment by conzett** [comment]
This is where I was thinking of putting it either way I guess, so we can merge it and I can reference it in the specs when I fix those up as part of #7 👍

**Review by conzett — APPROVED** [review]
LGTM 👍

**Pull request merged**
